docs/installation.rst

```diff
@@ -39,8 +39,14 @@ environments should consider using other packages that help manage them such as
 `pyenv-virtualenvwrapper <https://github.com/yyuu/pyenv-virtualenvwrapper>`_.
 
 
+Sources
+-------
+
+Girder can be installed from two sources, **pip** or **git repository**.
```

Comment:
I'm happy to make the change, but the difference is:

    pip install girder   # this downloads from pypi

vs.

    git clone git://github.com/girder/girder.git
    cd girder && pip install -e .

Both install modes use pip, but one installs from the local directory and one from the pypi package.